code/package.json

```diff
@@ -82,6 +82,7 @@
     "serve-storybooks": "http-server ./built-storybooks -p 8001",
     "smoketest-storybooks": "cross-env STORYBOOK_DISPLAY_WARNING=true DISPLAY_WARNING=true node -r esm ../scripts/smoketest-storybooks.js",
     "storybook:ui": "./lib/cli/bin/index.js dev --port 6006 --config-dir ./ui/.storybook --no-manager-cache",
+    "task": "cd .. && yarn task",
```

Comment:
I actually really like this change, but maybe @tmeasday wants to see this.

Reply:
Oh, sure, why not!

Reply:
Extracted in a separate PR